### DISCUSSION TITLE
I cannot access a public repository

### DISCUSSION THREAD

**Discussion by wajika** [community/community#23347] [New to GitHub]
<p>I can’t open this repository under my network, and a 404 appears.<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/a/9/a9b618b678dd5d42ff57e6dce03a8ef1ee560590.jpeg" data-download-href="/uploads/short-url/odkPCfwEi7iuEczi4nP8YHN9KqA.jpeg?dl=1" title="404" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/17058349/181076447-45085f94-fe92-4957-99f7-91dcc59d22fd.jpeg" alt="404" data-base62-sha1="odkPCfwEi7iuEczi4nP8YHN9KqA" width="690" height="312" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/a/9/a9b618b678dd5d42ff57e6dce03a8ef1ee560590_2_690x312.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/a/9/a9b618b678dd5d42ff57e6dce03a8ef1ee560590_2_1035x468.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/a/9/a9b618b678dd5d42ff57e6dce03a8ef1ee560590_2_1380x624.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/a/9/a9b618b678dd5d42ff57e6dce03a8ef1ee560590_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">404</span><span class="informations">1745×791 62.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>repository url:<br>
<a href="https://github.com/liu-xiao-guo/custom-apm-transactions" class="onebox" target="_blank" rel="noopener nofollow ugc">https://github.com/liu-xiao-guo/custom-apm-transactions</a></p>
<p>But other people can open this repository, and they showed me screenshots.</p><aside class="onebox allowlistedgeneric" data-onebox-src="https://ibb.co/y0zgPKB">
  <header class="source">
      <img src="https://user-images.githubusercontent.com/17058349/181076448-a90004ee-43f3-4eaf-98fc-98cf295cf762.png" class="site-icon" width="300" height="300">

      <a href="https://ibb.co/y0zgPKB" target="_blank" rel="noopener nofollow ugc">ImgBB</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/17058349/181076449-9b6ee0f1-4921-4ff9-93f2-98d6b40c8bf9.png" class="thumbnail onebox-avatar" width="180" height="180">

<h3><a href="https://ibb.co/y0zgPKB" target="_blank" rel="noopener nofollow ugc">github1</a></h3>

  <p>Image github1 hosted in ImgBB</p>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>Can anyone explain to me the reason?</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>The screenshot shows the repository is private (it says so next to the name), not public. So you won’t be able to see it unless specifically granted access.</p>

**Comment by wajika** [comment]
<aside class="quote no-group" data-username="wajika" data-post="1" data-topic="193717">
<div class="title">
<div class="quote-controls"></div>
 wajika:</div>
<blockquote>
<p>repository</p>
</blockquote>
</aside>
<p>ok thanks for your reply.</p>
